node/node.go

```diff
@@ -1110,12 +1095,14 @@ func (node *AlgorandFullNode) oldKeyDeletionThread() {
 		agreementProto := config.Consensus[hdr.CurrentProtocol]
 
 		node.mu.Lock()
-		node.accountManager.DeleteOldKeys(latestHdr, ccSigs, agreementProto)
+		node.accountManager.DeleteOldKeys(latestHdr, agreementProto)
 		node.mu.Unlock()
 
-		// PKI TODO: Maybe we don't even need to flush the registry.
-		// Persist participation registry metrics.
-		node.accountManager.FlushRegistry(2 * time.Second)
+		// Persist participation registry updates to last-used round and voting key changes.
+		err = node.accountManager.Registry().Flush(participationRegistryFlushMaxWaitDuration)
```

Comment:
I'm not sure this duration should be raised. Don't we want to keep the warning if it is taking > 2 seconds?

Reply:
It would be nice to get a warning about this specific flush timeout being slow, agreed. (As opposed to the REST API ones)